packages/fetch-cached-dns/index.js

```diff
@@ -1,84 +1,84 @@
-const { isIP } = require('net')
-const { format, parse } = require('url')
-const resolve = require('@zeit/dns-cached-resolve').default
-const { dnsCachedUrl } = require('./util')
+const { isIP } = require("net");
+const { format, parse } = require("url");
+const resolve = require("@zeit/dns-cached-resolve").default;
+const { dnsCachedUrl } = require("./util");
 
-module.exports = setup
+module.exports = setup;
 
-const isRedirect = v => ((v / 100) | 0) === 3
+const isRedirect = (v) => ((v / 100) | 0) === 3;
 
 function setup(fetch) {
   if (!fetch) {
-    fetch = require('node-fetch')
+    fetch = require("node-fetch");
   }
-  const { Headers } = fetch
+  const { Headers } = fetch;
 
   async function fetchCachedDns(url, opts) {
-    const parsed = parse(url)
-    const originalHost = parsed.host
-    const ip = isIP(parsed.hostname)
+    const parsed = parse(url);
+    const originalHost = parsed.host;
+    const ip = isIP(parsed.hostname);
     if (ip === 0) {
-      if (!opts) opts = {}
-      opts.headers = new Headers(opts.headers)
-      if (!opts.headers.has('Host')) {
-        opts.headers.set('Host', parsed.host)
+      if (!opts) opts = {};
+      opts.headers = new Headers(opts.headers);
+      if (!opts.headers.has("Host")) {
+        opts.headers.set("Host", parsed.host);
       }
-      opts.redirect = 'manual'
-      parsed.host = await resolve(parsed.hostname)
+      opts.redirect = "manual";
+      parsed.host = await resolve(parsed.hostname);
       if (parsed.port) {
-        parsed.host += `:${parsed.port}`
+        parsed.host += `:${parsed.port}`;
       }
-      url = format(parsed)
+      url = format(parsed);
     }
-    const res = await fetch(url, opts)
+    const res = await fetch(url, opts);
 
     // Update `res.url` to contain the original hostname instead of the IP address
-    res[dnsCachedUrl] = url
-    Object.defineProperty(res, 'url', {
+    res[dnsCachedUrl] = url;
+    Object.defineProperty(res, "url", {
       get() {
-        return parsed.href
-      }
-    })
+        return parsed.href;
+      },
+    });
 
     if (isRedirect(res.status)) {
-      const redirectOpts = Object.assign({}, opts)
-      redirectOpts.headers = new Headers(opts.headers)
+      const redirectOpts = { ...opts };
+      redirectOpts.headers = new Headers(opts.headers);
 
       // Per fetch spec, for POST request with 301/302 response, or any
       // request with 303 response, use GET when following redirect
       if (
         res.status === 303 ||
-        ((res.status === 301 || res.status === 302) && opts.method === 'POST')
+        ((res.status === 301 || res.status === 302) && opts.method === "POST")
       ) {
-        redirectOpts.method = 'GET'
-        redirectOpts.body = null
-        redirectOpts.headers.delete('content-length')
+        redirectOpts.method = "GET";
```

Comment:
Let's use single quotes like our other repos

Reply:
This should be enforced by the vercel/style-guide

Reply:
It is, but it seems to have been set up incorrectly.